avocado/plugins/remote.py

```diff
@@ -68,10 +68,10 @@ def configure(self, parser):
                                         help='Specify an identity file with '
                                         'a private key instead of a password '
                                         '(Example: .pem files from Amazon EC2)')
-        self.remote_parser.add_argument('--remote-no-copy',
-                                        dest='remote_no_copy',
+        self.remote_parser.add_argument('--remote-copy',
+                                        dest='remote_copy',
                                         action='store_true',
-                                        help="Don't copy tests and use the "
+                                        help="Copy tests and use the "
```

Comment:
I think all these `--copy` options should be documented as "Try to copy tests to...". There are many use-cases that will break with `--copy`, especially as we implement more advanced features, like multiplexing.

Or, like I would prefer, they should be removed, as they don't match the architecture of Avocado, where tests are not necessarily files. If removed, the documentation should provide an example of rsync invocation to copy tests, data-dirs, etc, which could be as simple as:

`$ cd avocado`
`$ rsync -a examples/tests -e ssh user@remote:~/avocado`
`$ avocado run --remote ...`

Or as a single line:

`$ rsync -a examples/tests -e ssh user@remote:~/avocado && avocado run --remote ...`